BraveWallet/Crypto/Stores/AssetDetailStore.swift

```diff
@@ -164,25 +167,41 @@ class AssetDetailStore: ObservableObject {
     keyring: BraveWallet.KeyringInfo,
     assetRatios: [String: Double]
   ) async -> [TransactionSummary] {
-    let network = await rpcService.network(.eth)
+    let coin = token.coin
+    let network = await rpcService.network(coin)
     let allTransactions = await withTaskGroup(of: [BraveWallet.TransactionInfo].self) { group -> [BraveWallet.TransactionInfo] in
       for account in keyring.accountInfos {
         group.addTask {
-          await self.txService.allTransactionInfo(.eth, from: account.address)
+          await self.txService.allTransactionInfo(coin, from: account.address)
         }
       }
       return await group.reduce([BraveWallet.TransactionInfo](), { partialResult, prior in
         return partialResult + prior
       })
     }
+    var solEstimatedTxFees: [String: UInt64] = [:]
+    if network.coin == .sol {
+      solEstimatedTxFees = await solTxManagerProxy.estimatedTxFees(for: allTransactions.map(\.id))
```

Comment:
same as here. but here we use `token.coin`

Reply:
Fixed in [700a537](https://github.com/brave/brave-ios/pull/5737/commits/700a537f6ca35a579f1e16f6cf526f262d9f1d03)